tools/Google.Cloud.Docs.Snippets/UpdateMaskSnippets.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.SecretManager.V1;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+
+namespace Google.Cloud.Tools.Snippets
```

Comment:
File scoped namespace (no curly brackets, see [tools/Google.Cloud.Docs.Snippets/LongRunningOperationsSnippets.cs](https://github.com/googleapis/google-cloud-dotnet/blob/main/tools/Google.Cloud.Docs.Snippets/LongRunningOperationsSnippets.cs)) for new files, here and in the others.